chef/data_bags/crowbar/bc-template-provisioner.json

```diff
@@ -75,6 +75,7 @@
       "use_serial_console": false,
       "serial_tty": "ttyS0,115200n8",
       "dhcp": {
+        "for-unknown-nodes": true,
```

Comment:
Apologies, I didn't see that yesterday, but we're avoiding adding new attributes with `-`, and always use `_` instead now.